src/lib.rs

```diff
@@ -325,6 +325,25 @@ impl Navigator {
         self.pwm.set_channel_off(channel.into(), value).unwrap();
     }
 
+    /// Like [`set_pwm_channel_value`](struct.Navigator.html#method.set_pwm_channel_value). This function
+    /// sets the Duty Cycle on a list of multiple channels.
```

Comment:
Maybe change the "on" to "for", meaning that the same value is applied to every channel of that list.